main.go

```diff
@@ -122,7 +130,10 @@ func main() {
 		go func() {
 			lnurlp, lnurlpResponse, err = GetJSON(GetJSONParams{url: lnurlpUrl, wg: &wg})
 			if err != nil {
-				e.Logger.Errorf("%v", err)
+				logger.WithFields(log.Fields{
+					"lightning_address": ln,
```

Comment:
The format of the keys is different to e.g. NWC. Does it matter?

Reply:
Probably doesn't matter that much